frontend/packages/dev-console/src/components/pipelineruns/logs/MultiStreamLogs.tsx

```diff
@@ -0,0 +1,132 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import * as classNames from 'classnames';
+import { saveAs } from 'file-saver';
+import { ContainerStatus, PodKind } from '@console/internal/module/k8s';
+import { LogControls, LoadingInline, FirehoseResult } from '@console/internal/components/utils';
+import { useFullScreen, useScrollDirection, ScrollDirection } from '@console/shared';
+import { containerToLogSourceStatus } from '../../../utils/pipeline-utils';
+import { getRenderContainers } from './logs-utils';
+import Logs from './Logs';
+import './MultiStreamLogs.scss';
+
+type LogsWrapperComponentProps = {
+  obj?: FirehoseResult<PodKind>;
```

Comment:
`obj?` can be undefined but you don't perform null checks in `LogsWrapperComponent`